src/glmtools.jl

```diff
@@ -45,26 +45,31 @@ linkfun(::SqrtLink, μ) = sqrt(μ)
 linkinv(::SqrtLink, η) = abs2(η)
 mueta(::SqrtLink, η) = 2η
 
+canonicallink(::Bernoulli) = LogitLink()
 canonicallink(::Binomial) = LogitLink()
 canonicallink(::Gamma) = InverseLink()
 canonicallink(::Normal) = IdentityLink()
 canonicallink(::Poisson) = LogLink()
 
 # For the "odd" link functions we evaluate the linear predictor such that mu is closest to zero where the precision is higher
-function glmvar(::Binomial, link::@compat(Union{CauchitLink,InverseLink,LogitLink,ProbitLink}), μ, η)
+function glmvar(::@compat(Union{Bernoulli,Binomial}), link::@compat(Union{CauchitLink,InverseLink,LogitLink,ProbitLink}), μ, η)
     μ = linkinv(link, ifelse(η < 0, η, -η))
     μ * (1 - μ)
 end
-glmvar(::Binomial, ::Link, μ, η) = μ * (1 - μ)
+glmvar(::@compat(Union{Bernoulli,Binomial}), ::Link, μ, η) = μ * (1 - μ)
 glmvar(::Gamma, ::Link, μ, η) = abs2(μ)
 glmvar(::Normal, ::Link, μ, η) = 1
 glmvar(::Poisson, ::Link, μ, η) = μ
 
+mustart(::Bernoulli, y, wt) = 0.5y + oftype(y, 0.25)
```

Comment:
Better write this as `y/2`. Cf. http://docs.julialang.org/en/latest/manual/style-guide/#avoid-using-floats-for-numeric-literals-in-generic-code-when-possible